### PR TITLE
Send email on integration tests fail

### PR DESCRIPTION
**Pull request by smiclea**

### CODE REVIEW

**Comment by claudiubelu** [comment]
We would want these integration tests to run periodically as well, and only send the email on the periodic job.

**Comment by claudiubelu** [comment]
Note that only the integration tests should be periodic, not the lint jobs.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -37,3 +37,16 @@ jobs:
       - name: Run integration tests
         run: |
           tox -vve integration
+
+      - name: Send email on failure
+        if: failure()
```
Comment (claudiubelu) [inline .github/workflows/ci.yaml]: we only want to send an email if the periodic job failed.

**Comment by smiclea** [comment]
@claudiubelu The PR has been updated. The workflow now runs periodically. The lint job has been excluded from running periodically and the email is only sent for integration testing in case of failure.

**Comment by claudiubelu** [comment]
LGTM!

The CI failures are unrelated, and they have already been fixed (see https://github.com/finos/legend-juju-bundle/pull/33).

**Pull request merged**